### DISCUSSION TITLE
Problems with Digital Ocean

### DISCUSSION THREAD

**Discussion by Caterina1994** [Budibase/budibase#1131] [Help] [answered]
Hey!
I already created an account on digital ocean .. but my web still performs on local host.
I followed the instructions but i dont think they are very clear.
Please can you help me?
thanks

**Comment by shogunpurple** [comment, marked as answer]
Hi @Caterina1994,

Thanks for taking the time to raise a discussion. Could you tell me which steps you have taken up to this point?

**Reply by Caterina1994** [reply]
Hi @shogunpurple . I made an account in Digital Ocean, Then a downloaded Docker + Docker Compose. I changed my ipv4 form Digital Ocean into de Host field and then deploy .. but it wont work. I tried with a lot of tutorials but dont seem to get around this

**Reply by shogunpurple** [reply]
Hi Caterina,

When you say it won't work, what is actually happening when you click deploy? Can you see an error message?

It seems to me that you may have installed docker and compose on your local machine, when you need to install it on the DigitalOcean droplet.

I think our one click DigitalOcean deployment might suit you better, as it will set everything up for you automatically.
https://marketplace.digitalocean.com/apps/budibase

Once that droplet is up and running, you should be able to point your builder to < droplets IP address >:10000.

**Comment by Caterina1994** [comment]
Hello Martin,

I actually did that and it wont work. you can see attached my images.
Thanks

**Comment by Caterina1994** [comment]
Nevermind. It works now :)
I was pressing preview instead of view deployed app.
I cant beleive i spend like 8 hours searching the error and install Docker
in my computer for nothing :) haha
So i dont need docker in my localthen? only digital ocean + budibase?
thanks

**Reply by shogunpurple** [reply]
Yep, you don't need docker on your local machine when you have a self hosted DigitalOcean instance.

**Comment by Caterina1994** [comment]
Please let me know if i need docker installed in my computer.
And now my csv wont work. It says i dont have Data and doesnt show but I do!

I tried to upload new csv but still doesnt work

**Reply by shogunpurple** [reply]
Hi Caterina, did you get your issue with your CSV resolved?
